web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -22,6 +22,10 @@
       (focusOut)="updateProjectTitle($event)"
       class="inline-editor"
     ></inline-editor>
+    <button mat-stroked-button (click)="onProjectsButtonClick()">
```

Comment:
I think this button was suggested by one of our community contributors, but it wasn't even part of the UX design. For now, we'd like to follow the pattern used by Docs and have clicking the Ground icon in the top left corner return to the project list.

Reply:
removed button and added to logo icon.